run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+WORKING_DIRECTORY=$(pwd)
+echo "Starting up in $WORKING_DIRECTORY"
+
+# try watching interval
+if [ "$1" == "" ]; then
+    INTERVAL="15s"
+else
+    INTERVAL=$1
+fi
+echo "Setting update interval to $INTERVAL"
+
+# start Flask API
+echo "Starting Flask..."
+flask run -p 80 -h 0.0.0.0 & sleep 1s
+
+# start Celery worker
+echo "Starting Celery worker..."
+celery -A spectre_analyses worker --loglevel=info -n divik-worker@%h &
+
+# watch them running
+while sleep $INTERVAL; do
+
+    ps aux | grep flask | grep -q -v grep
+    FLASK_DOWN=$?
+    if [ $FLASK_DOWN -ne 0 ]; then
+        echo "Flask died."
+        exit -1
+    fi
+
+    ps aux | grep celery | grep -q -v grep
+    CELERY_DOWN=$?
+    if [ $CELERY_DOWN -ne 0 ]; then
+        echo "Celery died."
+        exit -1
```

Comment:
Use `exit 2` for Celery to distinguish between failing processes easily.

Reply:
Accepted.